less/index.less

```diff
@@ -48,12 +46,154 @@ table {
 	border-spacing: 0;
 }
 
+//variables
+@import "variables.less";
+@standout: white;
+@font:'Roboto', sans-serif;
+@pfont:'Roboto', sans-serif;
+
+
+//mix-ins
+.bordered() {
+	border-bottom: 1px dashed #00b09b;
+}
+
+.background(){
+	background: linear-gradient(to right, #00b09b, #96c93d);
+}
 /* Set every element's box-sizing to border-box */
 * {
     box-sizing: border-box;
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	font-size: 62.5%;
+	line-height: 15px;
+	font-family: @font;
+}
+h1 { 
+	font-size: 3rem;
+	font-weight: bold;
+	@media @mobile{ 
+		margin-top: 20px;
+	} 
+}
+	
+	h2{ 
+	font-size: 2rem;
+	font-weight: bold;
+}
+	
+	h3 { 
+	font-size: 1.8rem;
+}
+
+
+
+
+header{ 
+	display: flex;
+	justify-content: space-between;
+	.background;
+	padding: 20px 20px 0 20px;
+  width: 100%;
+  height: auto; 
+	color: @standout;
+
+	
+	& .btn{ 
+    color: @standout;
+    border: none; 
+    cursor: pointer; 
+    background: none;
+		margin-bottom: 20px;
+		@media @mobile{ 
+			margin-bottom: 0;
+		} 
+	}
 }
 
+nav a{ 
+	color: @standout;
+	text-decoration: none;
+	font-size: 2rem;
+	 
+}
+
+.banner-content{ 
+	.background;
+	color: @standout;
+
+	& .title{ 
+		font-size: 3rem;
+		font-weight: bold;
+		margin-left: 40px;
+		padding: 10px;
+
+		@media @mobile{ 
+			text-align: center;
+			margin: 0 auto;
+		}
+	}
+}
+
+.avatar { 
+	width: 30%;
+	border-radius: 50%;
+	margin:0 0 10px 30px;
+	@media @mobile{ 
+		margin-left: 150px;
+	}
+	 }
+
+	 .main-content{ 
```

Comment:
Should be nested under body on the same level as header and the banner-content section. Then, you'd nest content-header h2 under main-content section. And then you'd nest the content-title under content-header.